docs/learning-powershell/README.md

```diff
@@ -51,15 +51,15 @@ You can use your favorite editor to write scripts.
 We use Visual Studio Code (VS Code) which works on Windows, Linux, and macOS.
 Click on the following link to create your first PowerShell script.
 
-- [Using Visual Studio Code (VS Code)][use-vscode-editor]
+- [Using Visual Studio Code (VS Code)](https://docs.microsoft.com/en-us/powershell/scripting/components/vscode/using-vscode?view=powershell-6)
 
 PowerShell Debugger
 ----
 
 Debugging can help you find bugs and fix problems in your PowerShell scripts.
 Click on the link below to learn more about debugging:
 
-- [Using Visual Studio Code (VS Code)][use-vscode-debugger]
+- [Using Visual Studio Code (VS Code)](https://docs.microsoft.com/en-us/powershell/scripting/components/vscode/using-vscode?view=powershell-6#debugging-with-visual-studio-code)
```

Comment:
Please remove the `[use-vscode-debugger]:./using-vscode.md#debugging-with-vs-code` line.